api/CookieChangeEvent.json

```
@@ -0,0 +1,197 @@
+{
+  "api": {
+    "CookieChangeEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CookieChangeEvent",
+        "support": {
+          "chrome": {
+            "version_added": "87"
+          },
+          "chrome_android": {
+            "version_added": "87"
+          },
+          "edge": {
+            "version_added": "87"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
```

```suggestion
          "opera": {
            "version_added": "73"
          },
          "opera_android": {
            "version_added": "62"
          },
```

This is mirroring Chromium 87, and should be possible to apply throughout.

---

updated in recent commit.